docs/cluster-configuration.md

```diff
@@ -18,17 +18,17 @@ cluster = Cluster(ClusterConfiguration(
     min_memory=2, # Default 2
     max_memory=2, # Default 2
     num_gpus=0, # Default 0
-    mcad=True, # Default True
+    appwrapper=True, # Default False
```

Comment:
I'd suggest we remove that parameter at the moment from the example, but keep the description below.

Reply:
I elided app wrappers entirely from this document and chopped the description below (which was also incorrect; whether there is an appwrapper or not, the behavior of the KubeRay and Codeflare operators are identical).